hera_cal/io.py

```diff
@@ -1366,22 +1415,25 @@ def _adapt_metadata(self, info_dict, skip_lsts=False):
         info_dict['data_antpos'] = {ant: info_dict['antpos'][ant] for ant in info_dict['data_ants']}
         info_dict['times'] = np.unique(info_dict['times'])
         info_dict['times_by_bl'] = {ap: info_dict['times'] for ap in info_dict['antpairs']}
-        info_dict['times_by_bl'].update({(a2, a1): info_dict['times'] for (a1, a2) in info_dict['antpairs']})
+        # info_dict['times_by_bl'].update({(a2, a1): info_dict['times'] for (a1, a2) in info_dict['antpairs']})
```

Comment:
why was this removed?

Reply:
It wasn't there originally -- I added it when I thought I needed to manually index all the conjugated bls. I'll remove it completely.